doc/source/report/behavior.rst

```diff
@@ -50,165 +50,70 @@ to capture the behavioral profile:
+   -  **Drinking** Bout Size: ``Total Drinking Amount/ Number of Bouts``
+      (mg/bout)
+      -  **Drinking** Bout Duration:
+         ``Total Drinking Time/ Number of Bouts`` (s/bout)
+      -  **Drinking** Bout Intensity:
+         ``Drinking Amount/ Total Drinking Time`` (mg/s)
+         -  **Drinking** Bout Event Rate:
+            ``Number of Events/ Total Drinking Time`` (events/s)
+         -  **Drinking** Event Size:
+            ``Total Drinking Amount/ Number of Events`` (mg/event)
 
--  AS [*Drinking \| Feeding \| Locomotion*\ ] Intensity
--  *A note about intensity:* We are not entirely sure what the Tecott
-   Lab's meaning of "intensity" is. Our current hypothesis is that
-   intensity is defined as quantity over active state time. E.g. for
-   drinking, intensity is the quantity consumed divided by the total
-   amount of time the mouse is in an active state.
+Data Collection:
+----------------
 
-   -  [*Drinking \| Feeding \| Locomotion*\ ] Bout Size
+Our underlying functions depend on the following key data
+requirements for each **mouse, strain and day**:
 
-      -  [*Drinking \| Feeding \| Locomotion*\ ] Bout Duration
-      -  [*Drinking \| Feeding \| Locomotion*\ ] Bout Intensity
+- Active State
+- Inactive State
+- Moving Active State
+- Moving Inactive State
+- Total Distance Travelled (meters)
+- Food Consumption (grams)
+- Water Consumption (milligrams)
 
-         -  [*Drinking \| Feeding \| Locomotion*\ ] Bout EventRate
-         -  [*Drinking \| Feeding \| Locomotion*\ ] Event Size
+These data requirements are sourced via the ``mousestyles`` ``data
+loader`` utilities.
 
-   -  [*Drinking \| Feeding \| Locomotion*\ ] Bout Rt
+The Food and Water consumption data was only provided on a daily
+basis and our ``behavior`` utilities assume that such quantities
+are uniformly consumed over the time period studied.
 
-Additionally the following needs to be calculated for Inactive /Active
-State:
+Illustrative Examples
+---------------------
 
-- AS Probability: the probability of AS among all time period
-- AS Duration: the length of time of AS
-- AS Rate: the reciprocal of average length of time of AS
+We illustrate the use of the ``behavior`` utilties with a
+motivating research question:
 
-Data Collection:
-----------------
+*How do do the key feeding metrics compare across 2 different mice
+for the entire 11 days?*
 
-The data we have:
+In this code, we see a few important features. First, we create
+behavior trees using the ``compute_tree`` function, and demonstrate
+their pretty-printing functionality.  Then, we merge lists of trees
+into one larger tree of lists, which we can then summarize or turn
+into a Pandas data frame. Finally, we show how to plot the results
+using ``matplotlib``.
 
-- the observations of location for each mice, $(x, y, t)$ with $t$ small.
-- the aggregated time binned features about each event and its intensity.
+.. literalinclude:: report/plots/behavior_example.py
```

Comment:
I think you can do something like:

```
To also include the source code for the plot in the document, pass the include-source parameter:

.. plot:: pyplots/ellipses.py
   :include-source:
```

I am using modified code from matplotlib, so most of this should be true:
http://matplotlib.org/sampledoc/extensions.html

I submitted the changes upstream as well:
https://github.com/matplotlib/matplotlib/pull/6327